src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -377,50 +381,60 @@ public ColibriConferenceIQ createColibriChannels(
     private void maybeThrowOperationFailed(Packet response)
         throws OperationFailedException
     {
-        if (response == null)
-        {
-            allocChannelsErrorCode
-                = OperationFailedException.NETWORK_FAILURE;
-            allocChannelsErrorMsg
-                = "Failed to allocate colibri channels: response is null."
-                + " Maybe the response timed out.";
-        }
-        else if (response.getError() != null)
+        // This code block must be protected, because in the last "if" a
+        // decision is made based on the value of allocChannelsErrorCode, which
+        // could have been written by another thread. We use syncRoot, because
+        // the existing code already runs with its lock acquired.
```

Comment:
I don't understand this part of the comment:
> We use syncRoot, because the existing code already runs with its lock acquired.
It is being acquired here, isn't it ? It was released in top level, before calling this method.